### PR TITLE
:bug: repeated search works

### LINKED ISSUE #4490: Repeated Search doesn't work
### What happened?

Search works only first time, second time it doesn't. Chain agnostic problem.

### Please reproduce in steps

1. try searching for random word
2. try searching for another word

### Expected Behavior

I'd expect this to work.

### What browsers are you seeing the problem on?

Chrome

### At which address did you encounter bug?

beta.kodadot.xyz, kodadot.xyz

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4490
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/73316633/207708892-9cb7b394-e088-4fef-829f-a51789820ac5.mp4

### CODE REVIEW

File: components/search/Search.vue [hunk 1]
```diff
@@ -243,7 +243,7 @@ export default class Search extends mixins(
   @Emit('update:search')
   @Debounce(50)
   updateSearch(value: string): string {
-    if (!value != !this.$route.query.search && value !== this.searchQuery) {
+    if (value != this.$route.query.search && value !== this.searchQuery) {
```
Comment (Jarsen136) [inline components/search/Search.vue:246]: how about using `!==` ?

**Review by Jarsen136 — COMMENTED** [review]
small stuff

**Comment by yangwao** [comment]
pay 40 usd

**Comment by petersopko** [comment]
> 😍 Perfect, I’ve sent the payout 💵 $40 @ 26.53 USD/KSM ~ 1.508 $KSM 🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y) 🔗 [0x3971cbebc412e83140008893e6f33a6b29c11c11e6f7e796ff169eb0c536561e](https://kusama.subscan.io/extrinsic/0x3971cbebc412e83140008893e6f33a6b29c11c11e6f7e796ff169eb0c536561e)
> 
> 🪅 Let’s grab another issue and get rewarded! 🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

this one didn't work 👀

**Comment by yangwao** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 25.2 USD/KSM ~ 1.587 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0xa15007fb3fd7519a1132f26ad6ba4b6bddd8fceb85c5c84c0039d952eb9bd3b1](https://kusama.subscan.io/extrinsic/0xa15007fb3fd7519a1132f26ad6ba4b6bddd8fceb85c5c84c0039d952eb9bd3b1)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**